docs/_reporting/power-bi/README.md

```diff
@@ -57,67 +57,71 @@ Microsoft offers several ways to analyze and report on your cloud costs. For qui
+| Billing profiles                                    |                ✅                 |                ✅                 |             ✅                     |              ✅               |
+| Invoice sections                                    |                ✅                 |                ✅                 |             ✅                     |              ✅               |
+| CSP customers (partner only)                        |                ✅                 |                ✅                 |             ✅                     |              ✅               |
+| Management groups                                   |                ❌                 |                ❌                 |             ✅                     |              ❌               |
+| Subscriptions                                       |                ✅                 |                ✅                 |             ✅                     |              ✅               |
+| Resource groups                                     |                ✅                 |                ✅                 |             ✅                     |              ✅               |
+| Supports savings plans<sup>4</sup>                  |                ✅                 |                ✅                 |             ✅                     |              ✅               |
+| Supports savings plan recommendations               |                ❌                 |                🔜                 |             ✅                     |              🔜               |
+| Supports multiple scopes                            |                ✅                 |                ✅                 |             ✅                     |              ✅               |
+| Supports scopes in different tenants<sup>5</sup>    |          ⚠️<sup>5</sup>           |          ⚠️<sup>5</sup>           |             ❌                     |          ⚠️<sup>5</sup>(via Hubs) |
+| Faster data load times                              |                🔜                 |                ✅                 |             ✅                     |         ✅ (via Hubs)         |
+| Supports >$65M in cost details                      |                ❌                 |                ✅                 |             ❌                     |              ✅               |
+| Accessible outside of Power BI                      |                ✅                 |                ✅                 |             ✅                     |              ✅               |
+| Kusto Query Language (KQL) support                        |                ❌                 |                ❌                 |             ✅                     |              ❌               |
+| Native integration with cost optimization workbook |         ❌                 |                ❌                 |             ✅<sup>6</sup>                     |              ❌               |
```

Comment:
Should we expand this to workbooks in general? Then that's possible today. It's just not something we offer. Beyond that, I'm not sure the COW is big enough to really call out as a feature someone's going to use to make a decision 🤔
```suggestion
| Native integration with cost optimization workbook |         ❌                 |                ❌                 |             🔜                     |              ❌               |
```